### PR TITLE
update script to only publish istio-sidecar.deb

### PR DESCRIPTION
**Pull request by rkpagadala**

### CODE REVIEW

**Comment by mattdelco** [comment]
/lgtm
It might also be worthwhile to update the release notes URL to point to something like https://istio.io/about/notes .  Wiki entries weren't created for 0.5 nor 0.5.1 and I doubt there will be [m]any in the future.  Is there a version of istio newer than 0.2.12 that could replace it as the latest stable release?

**Comment by mattdelco** [comment]
/lgtm
Well, actually I have some reservations but this is the default template so I'm not going to be that picky.  The build uses version x.y.z, but so far the existing webpages use just x.y.  This difference, plus the delay in hand-crafting the release notes, are factors why I had considered using a version-less default.  However, if somebody can set up a rule on istio.io so that any unknown page requested under notes/... just redirects to notes/ then that might be the most ideal.

File: release/create_github_release.sh [hunk 1]
```diff
@@ -88,7 +88,7 @@ cat << EOF > ${REQUEST_FILE}
 {
   "tag_name": "${VERSION}",
   "target_commitsh": "${SHA}",
-  "body": "[ARTIFACTS](http://gcsweb.istio.io/gcs/istio-release/releases/${VERSION}/)\\n* [istio-agent](https://storage.googleapis.com/istio-release/releases/${VERSION}/deb/istio-agent.deb)\\n* [istio-auth-node-agent](https://storage.googleapis.com/istio-release/releases/${VERSION}/deb/istio-auth-node-agent.deb)\\n* [istio-proxy](https://storage.googleapis.com/istio-release/releases/${VERSION}/deb/istio-proxy.deb)\\n\\n[RELEASE NOTES](http://github.com/istio/istio/wiki/v${VERSION})",
+  "body": "[ARTIFACTS](http://gcsweb.istio.io/gcs/istio-release/releases/${VERSION}/)\\n* [istio-sidecar.deb](https://storage.googleapis.com/istio-release/releases/${VERSION}/deb/istio-sidecar.deb)\\n\\n[RELEASE NOTES](https://istio.io/about/notes/${VERSION}.html)",
```
Comment (ldemailly) [inline release/create_github_release.sh]: that VERSION isn't the same as this VERSION
(0.6.0 vs 0.6)

(and unrelated I still think the wiki is better for raw notes, specially better than nothing but I guess that's OT)

**Pull request merged**